### ISSUE TITLE
Possibility to give name to the Room.

### ISSUE THREAD

**Issue by gk-patel** [freeboardgames/FreeBoardGames.org#648]
First of all I really like the idea of giving names to the rooms, and the random names given by the system are also very good.  
Nevertheless, it would be good to have a possibility to give our own name to the room. 
For example, this is good alternative for not having to a share link, just give a pre-defined room-name and ask friends to jump in. (please remember, I am talking about friends in-person, eg. sitting on the couch together, etc.) 

Maybe what can be done is, when someone creates a room he gets a suggested name for the room, if he does not like it, he can change it.

**Comment by flamecoals** [comment]
This was intentional, we were worried about bad words, curses, etc on the home page. However, I think we could try to implement some sort of "bad word list"  and allow users to specify names.

**Comment by flamecoals** [comment]
the "bad word list"  should be useful for chat too, maybe even required. And I will be focusing on chat now.

**Comment by larry801** [comment]
"bad word list" is not very useful for avoiding curses or spammer, it is very easy to workaround such as  master -> mastor, such as annoying wordpress spam comments.
QRCode is suitable for sharing with a friend offline.

**Comment by flamecoals** [comment]
@larry801 agreed. However, the intention of the public rooms feature was for users to be able to play with strangers. I don't think this have catched up so far, even though we have 20ish users sometimes online in real time on the website, I barely see them using this feature. I think that having the chat (maybe even on the home page) will help with that. But we might want to consider a "play now" feature where we do the match making for the users.

**Comment by larry801** [comment]
@flamecoals  Chat on home page has similar caveat, private chat in room is a lot better

**Comment by gk-patel** [comment]
I agree Chat on Home Page can be prone to abuZe ;) 
If we can garuentee that the names for open rooms dont repeate - or have a low probability of repetation, then we dont need to have custom name. I completely agree with your view. 
If you think that pepbability of repetation is very low, then you can close this. 
Thanks for consideration.

**Comment by gk-patel** [comment]
So, after doing a bit rethinking, the original idead for this issue was to garuentee that we can direct friends to a unique room, so if the random names can somehow be added to the url that would be okay, 

so instead of
https://www.freeboardgames.org/room/44kMxSHUm
something like this
https://www.freeboardgames.org/room/starfish

**Comment by thgh** [comment]
Hi, jitsi has this feature: e.g. https://meet.jit.si/myboardgameroom
This allows to have a fixed "meeting spot" which is pretty nice!
